CODEOWNERS

```
@@ -8,7 +8,7 @@
 /charts/simple-nginx/ @douglascamata
 
 # Logging chart
-/charts/newrelic-logging/ @jsubirat @matildasmeds @tejunior
+/charts/newrelic-logging/ @jsubirat @danybmx @jasonrclark 
```

Is it ok to ping users directly instead of the team?

```suggestion
/charts/newrelic-logging/ @newrelic/logging 
```